skymp5-functions-lib/src/utils/functionInfo.ts

```diff
@@ -2,40 +2,17 @@ export class FunctionInfo<F extends { toString: () => string }> {
   public constructor(private f: F) {}
 
   get text(): string {
-    return `
-        try {
-          ${this.getTextWithoutErrorHandling()}
-        } catch(err) {
-          ctx.sp.printConsole('[CTX ERROR]', err, '\\n', ${this.f});
-        }`;
+    return `try{${this.getTextWithoutErrorHandling()}}catch(e){` +
+        `ctx.sp.printConsole('[CTX ERROR]', e, '\\n', ${this.f})}`
   }
 
   getText(args?: Record<string, unknown>): string {
-    let result = this.text;
-
-    for (const name in args) {
-      switch (typeof args[name]) {
-        case 'number':
-          result = `const ${name} = ${args[name]};\n` + result;
-          break;
-        case 'string':
-          result = `const ${name} = '${args[name]}';\n` + result;
-          break;
-        case 'boolean':
-          result = `const ${name} = ${args[name]};\n` + result;
-          break;
-        case 'object':
-          if (Array.isArray(args[name])) {
-            result = `const ${name} = [${args[name]}];\n` + result;
-          }
-          break;
-        case 'function':
-          result = `const ${name} = ${(args[name] as Function).toString()};\n` + result;
```

Comment:
I don't know if it is actually needed by anyone... noticed this only now. Maybe it's ok to just remove this